### PR TITLE
[#12048] Get Account and Accounts actions

### PR DESCRIPTION
**Pull request by domlimm**
Part of #12048.

**Outline of Solution**

This PR consists of the following changes:
- Update GetAccountAction
- Update GetAccountsAction
- Introduce new methods to support above Actions

### CODE REVIEW

File: src/main/java/teammates/ui/webapi/GetAccountAction.java [hunk 1]
```diff
@@ -13,12 +13,13 @@ class GetAccountAction extends AdminOnlyAction {
     public JsonResult execute() {
         String googleId = getNonNullRequestParamValue(Const.ParamsNames.INSTRUCTOR_ID);
 
-        AccountAttributes accountInfo = logic.getAccount(googleId);
-        if (accountInfo == null) {
+        Account account = sqlLogic.getAccountByGoogleId(googleId);
```
Comment (samuelfangjw) [inline src/main/java/teammates/ui/webapi/GetAccountAction.java:16]: Need to support dual db for this. Account has an isMigrated flag you need to check.

Comment (domlimm) [reply src/main/java/teammates/ui/webapi/GetAccountAction.java]: Have made this change!

**Comment by domlimm** [comment]
@samuelfangjw Samuel, for `GetAccountsAction` do I need to check for migrated accounts too? If so, how should I do that? As we're retrieving all accounts associated with the given email.

**Comment by samuelfangjw** [comment]
> @samuelfangjw Samuel, for `GetAccountsAction` do I need to check for migrated accounts too? If so, how should I do that? As we're retrieving all accounts associated with the given email.

Retrieve from both DBs and combine the output.

File: src/it/java/teammates/it/storage/sqlapi/AccountsDbIT.java [hunk 2]
```diff
@@ -16,6 +18,38 @@ public class AccountsDbIT extends BaseTestCaseWithSqlDatabaseAccess {
...
+        accountsDb.createAccount(account1);
+        accountsDb.createAccount(account2);
+        accountsDb.createAccount(account3);
+
+        accounts = accountsDb.getAccountsByEmail(email);
+
+        assertEquals(3, accounts.size());
+        verifyEquals(account1, accounts.get(0));
+        verifyEquals(account2, accounts.get(1));
+        verifyEquals(account3, accounts.get(2));
```
Comment (samuelfangjw) [inline src/it/java/teammates/it/storage/sqlapi/AccountsDbIT.java:50]: API makes no assumptions about ordering of return values.

**Review by samuelfangjw — CHANGES_REQUESTED** [review]

**Review by samuelfangjw — APPROVED** [review]

**Pull request merged**